### PR TITLE
2.0.0 Release Notes

### PR DESCRIPTION
**Pull request by svarnau**
Update website with release notes for 2.0.0 and 2.0.1 (up coming).

Re-worked the download page to point to the provisioning guides, rather than try to summarize installation in a release independent way. 

Use apache download guidelines to point to mirror sites for tar-file downloads, but the main apache site for signatures and checksums.

Place-holder for 2.0.1 release download since it is not official yet.

### CODE REVIEW

File: docs/src/site/markdown/download.md [hunk 1]
```diff
@@ -12,144 +12,54 @@
...
+* [Release Notes](release-notes-2-0-1.html)
+* [Documentation](documentation.html#20x_Releases)
 
-Perform this step as the **`trafodion`** user.
+## 2.0.0
 
-First, check the status of the Trafodion environment.
+* [Release Notes](release-notes-2-0-0.html)
+* [Source Code Release][src200]  -  [PGP][pgp200] [MD5][md5200] [SHA1][sha200]
+* Convenience Binaries
```
Comment (zellerh) [inline docs/src/site/markdown/download.md]: Should this mention that there is no client package in 2.0, so people don't search for it unnecessarily? Maybe something like a short note: Release 2.0.0 does not include convenience binaries for the ODBC/JDBC clients. Those must be built from source, if needed.

**Comment by zellerh** [comment]
Looks good to me, just wondering whether we need a note about the missing client package.

File: docs/src/site/markdown/download.md (hunk 1 above)
Comment (svarnau) [inline docs/src/site/markdown/download.md]: Good idea. I'll add it.

**Comment by zellerh** [comment]
Thanks for adding the note.

**Comment by sureshsubbiah** [comment]
Looks good. +1

**Comment by DaveBirdsall** [comment]
Looks good to me too.

**Pull request merged**